projects/mysql-connector-j/ServerLoginFuzzer.java

```diff
@@ -0,0 +1,25 @@
+import com.code_intelligence.jazzer.api.FuzzedDataProvider;
```

Comment:
can you add license headers on all the .java files?

Reply:
@aschaich

Reply:
done